SpatialGDK/Extras/schema/component_presence.schema

```diff
@@ -1,13 +1,13 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
 package unreal;
 
-// The ComponentPresence component should be present on all entities that 
-// should be load-balanced. This contains a single property which is the 
-// list of component IDs that are present on the entity. Currently, this 
-// is used for enabling dynamic components in a multi-worker environment,
-// and should be useful in future for deducing entity completeness without
-// critical sections.
+// The ComponentPresence component should be present on all entities.
```

Comment:
Shouldn't this comment change be reverted?

Reply:
No, the old comment is out-of-date and refers to component presence being on all load-balancing entities, instead, it's now on everything.